### PR TITLE
[TIMOB-20174] iOS: Moving properties from iPhone nameSpace to iOS

### PR DESCRIPTION
**Pull request by AngelkPetkov**
[JIRA](https://jira.appcelerator.org/browse/TIMOB-20174?filter=-1)

### CODE REVIEW

**Comment by hansemannn** [comment]
Whats the status of this PR?

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 1]
```diff
@@ -20,7 +23,10 @@ properties:
         and [NONE](Titanium.UI.iPhone.AnimationStyle.NONE).
     type: Number
     permission: read-only
-    
+    deprecated:
+        since: 6.0.0
+        notes: Use Titanium.UI.iOS.AnimationStyle.CURL_DOWN instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Missing link e.g. `[Titanium.UI.iOS.AnimationStyle.CURL_DOWN](Titanium.UI.iOS.AnimationStyle.CURL_DOWN)`

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 2]
```diff
@@ -34,7 +40,10 @@ properties:
         and [NONE](Titanium.UI.iPhone.AnimationStyle.NONE).
     type: Number
     permission: read-only
-    
+    deprecated:
+        since: 6.0.0
+        notes: Use Titanium.UI.iOS.AnimationStyle.CURL_UP instead
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Missing link e.g. `[Titanium.UI.iOS.AnimationStyle.CURL_UP](Titanium.UI.iOS.AnimationStyle.CURL_UP)`

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 3]
```diff
@@ -48,7 +57,10 @@ properties:
         and [NONE](Titanium.UI.iPhone.AnimationStyle.NONE).
     type: Number
     permission: read-only
-    
+    deprecated:
+        since: 6.0.0
+        notes: Use Titanium.UI.iOS.AnimationStyle.FLIP_FROM_LEFT instead
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Missing link e.g. `[Titanium.UI.iOS.AnimationStyle.CURL_LEFT](Titanium.UI.iOS.AnimationStyle.CURL_LEFT)`

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 4]
```diff
@@ -62,7 +74,10 @@ properties:
         and [NONE](Titanium.UI.iPhone.AnimationStyle.NONE).
     type: Number
     permission: read-only
-    
+    deprecated:
+        since: 6.0.0
+        notes: Use Titanium.UI.iOS.AnimationStyle.FLIP_FROM_RIGHT instead
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Missing link e.g. `[Titanium.UI.iOS.AnimationStyle.CURL_RIGHT](Titanium.UI.iOS.AnimationStyle.CURL_RIGHT)`

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 5]
```diff
@@ -76,3 +91,6 @@ properties:
         and [NONE](Titanium.UI.iPhone.AnimationStyle.NONE).
     type: Number
     permission: read-only
+    deprecated:
+        since: 6.0.0
+        notes: Use Titanium.UI.iOS.AnimationStyle.NONE instead
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Missing link e.g. `[Titanium.UI.iOS.AnimationStyle.NONE](Titanium.UI.iOS.AnimationStyle.NONE)`

File: apidoc/Titanium/UI/iPhone/ListViewStyle.yml [hunk 6]
```diff
@@ -10,6 +10,9 @@ excludes:
 platforms: [iphone, ipad]
 since: "3.1.0"
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.ListViewStyle](Titanium.UI.iOS.ListViewStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/ListViewStyle.yml]: Consistent naming `Titanium.`.

File: apidoc/Titanium/UI/iPhone/NavigationGroup.yml [hunk 7]
```diff
@@ -2,7 +2,7 @@
 name: Titanium.UI.iPhone.NavigationGroup
 deprecated:
     since: "3.1.3"
-    removed: "3.2.0"
+    removed: "6.0.0"
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/NavigationGroup.yml]: It's already removed in 3.2.0, we dont need to touch the whole proxy.

File: apidoc/Titanium/UI/iPhone/ProgressBarStyle.yml [hunk 8]
```diff
@@ -6,6 +6,9 @@ extends: Titanium.Proxy
 platforms: [iphone, ipad]
 since: "0.8"
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.ProgressBarStyle](Titanium.UI.iOS.ProgressBarStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/ProgressBarStyle.yml]: Consistent naming `Titanium`.

File: apidoc/Titanium/UI/iPhone/RowAnimationStyle.yml [hunk 9]
```diff
@@ -5,6 +5,9 @@ extends: Titanium.Proxy
 platforms: [iphone, ipad]
 createable: false
 since: "0.9"
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.RowAnimationStyle](Titanium.UI.iOS.RowAnimationStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/RowAnimationStyle.yml]: Consistent naming `Titanium`.

File: apidoc/Titanium/UI/iPhone/ScrollIndicatorStyle.yml [hunk 10]
```diff
@@ -5,6 +5,9 @@ extends: Titanium.Proxy
 since: "0.9"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.ScrollIndicatorStyle](Titanium.UI.iOS.ScrollIndicatorStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/ScrollIndicatorStyle.yml]: Consistent naming `Titanium`.

File: apidoc/Titanium/UI/iPhone/SystemButton.yml [hunk 11]
```diff
@@ -33,6 +33,9 @@ extends: Titanium.Proxy
 since: "0.8"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.SystemButton](Titanium.UI.iOS.SystemButton) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/SystemButton.yml]: Consistent naming `Titanium`.

File: apidoc/Titanium/UI/iPhone/SystemButtonStyle.yml [hunk 12]
```diff
@@ -38,6 +38,9 @@ extends: Titanium.Proxy
 since: "0.8"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.SystemButtonStyle](Titanium.UI.iOS.SystemButtonStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/SystemButtonStyle.yml]: Consistent naming `Titanium`.

File: apidoc/Titanium/UI/iPhone/SystemButtonStyle.yml [hunk 13]
```diff
@@ -50,7 +53,10 @@ properties:
         color to show through.
     type: Number
     permission: read-only
-    
+    deprecated:
+        since: 3.4.2
+        removed: "3.60"
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/SystemButtonStyle.yml]: Spelling `"3.6.0"` instead of `"3.60"` and `3.4.2` in quotes (`"3.4.2"`).

File: apidoc/Titanium/UI/iPhone/StatusBar.yml [hunk 14]
```diff
@@ -5,6 +5,9 @@ extends: Titanium.Proxy
 since: "0.9"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/StatusBar.yml]: Missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/ListViewScrollPosition.yml [hunk 15]
```diff
@@ -7,6 +7,9 @@ summary: |
 extends: Titanium.Proxy
 since: '3.1.0'
 platforms: [iphone, ipad]
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.ListViewScrollPosition](Titanium.UI.iOS.ListViewScrollPosition) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/ListViewScrollPosition.yml]: Consistent naming `Titanium`, missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/ListViewCellSelectionStyle.yml [hunk 16]
```diff
@@ -11,6 +11,9 @@ excludes:
     methods: [addEventListener,applyProperties,fireEvent,removeEventListener]
     properties: [bubbleParent]
 createable: false
+deprecated:
+    since: 6.0.0
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/ListViewCellSelectionStyle.yml]: Missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 17]
```diff
@@ -5,6 +5,9 @@ extends: Titanium.Proxy
 platforms: [iphone, ipad]
 since: "0.9"
 createable: false
+deprecated:
+    since: 6.0.0
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/AnimationStyle.yml [hunk 18]
```diff
@@ -5,6 +5,9 @@ extends: Titanium.Proxy
...
+    notes: Use [Ti.UI.iOS.AnimationStyle](Titanium.UI.iOS.AnimationStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AnimationStyle.yml]: Consistent `Titanium`.

File: apidoc/Titanium/UI/iPhone/AlertDialogStyle.yml [hunk 19]
```diff
@@ -7,6 +7,9 @@ extends: Titanium.Proxy
 since: "3.0.0"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.AlertDialogStyle](Titanium.UI.iOS.AlertDialogStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/AlertDialogStyle.yml]: Consistent `Titanium`, missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/TableViewCellSelectionStyle.yml [hunk 20]
```diff
@@ -7,6 +7,9 @@ extends: Titanium.Proxy
 since: "0.9"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.TableViewCellSelectionStyle](Titanium.UI.iOS.TableViewCellSelectionStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/TableViewCellSelectionStyle.yml]: Consistent naming `Titanium`, missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/TableViewScrollPosition.yml [hunk 21]
```diff
@@ -7,6 +7,9 @@ extends: Titanium.Proxy
 since: "0.9"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.TableViewScrollPosition](Titanium.UI.iOS.TableViewScrollPosition) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/TableViewScrollPosition.yml]: Consistent naming `Titanium`, missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/TableViewStyle.yml [hunk 22]
```diff
@@ -7,6 +7,9 @@ extends: Titanium.Proxy
 platforms: [iphone, ipad]
 since: "0.9"
 createable: false
+deprecated:
+    since: 6.0.0
+    notes: Use [Ti.UI.iOS.TableViewStyle](Titanium.UI.iOS.TableViewStyle) instead.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/TableViewStyle.yml]: Consistent naming `Titanium`, missing quotes in "6.0.0".

File: apidoc/Titanium/UI/iPhone/iPhone.yml [hunk 23]
```diff
@@ -179,11 +179,17 @@ properties:
   - name: appBadge
     summary: Value of the badge for the application's springboard icon.
     type: Number
-    
+    deprecated:
+        since: "6.0.0"
+        notes: Use Ti.UI.iOS.appBadge instead please.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/iPhone.yml]: Consistent naming `Titanium`, missing link (see above).

File: apidoc/Titanium/UI/iPhone/iPhone.yml [hunk 24]
```diff
@@ -179,11 +179,17 @@ properties:
...
+
   - name: appSupportsShakeToEdit
     summary: Determines whether the shake to edit system-wide capability is enabled.
     type: Boolean
-    
+    deprecated:
+        since: "6.0.0"
+        notes: Use Ti.UI.iOS.appSupportsShakeToEdit instead please.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/iPhone.yml]: Consistent naming `Titanium`, missing link (see above).

File: apidoc/Titanium/UI/iPhone/iPhone.yml [hunk 25]
```diff
@@ -191,7 +197,11 @@ properties:
         Use the [fullscreen](Titanium.UI.Window.fullscreen) property of the window to control if the staus bar is shown or not.
     type: Boolean
     permission: read-only
-    
+    deprecated:
+        since: "3.1.3"
+        notes: Use Ti.UI.windows.fullscreen instead please.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/iPhone.yml]: Consistent naming `Titanium`, missing link (see above).

File: apidoc/Titanium/UI/iPhone/iPhone.yml [hunk 26]
```diff
@@ -202,7 +212,10 @@ properties:
                  Titanium.UI.iPhone.StatusBar.GREY, Titanium.UI.iPhone.StatusBar.LIGHT_CONTENT,
                  Titanium.UI.iPhone.StatusBar.OPAQUE_BLACK, Titanium.UI.iPhone.StatusBar.TRANSLUCENT_BLACK ]
     permission: read-only
-
+    deprecated:
+        since: "3.1.3"
+        notes: Use Titanium.UI.Window.statusBarStyle instead please.
```
Comment (hansemannn) [inline apidoc/Titanium/UI/iPhone/iPhone.yml]: Missing link (see above), remove the "please". Also check that one in all other places.

File: iphone/Classes/TIUIiOSAlertDialogStyleProxy.h [hunk 27]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2012 by Appcelerator, Inc. All Rights Reserved.
```
Comment (hansemannn) [inline iphone/Classes/TIUIiOSAlertDialogStyleProxy.h]: Copyright of all new files to 2016. Check all other new files you created as well.

File: iphone/Classes/TiUIiOSAnimationStyleProxy.h [hunk 28]
```diff
@@ -0,0 +1,25 @@
...
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2010 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIIOSANIMATIONSTYLE
+
+#import "TiProxy.h"
+
+
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSAnimationStyleProxy.h]: Multiple line-breaks, check other created files as well.

File: iphone/Classes/TiUIiOSProgressBarStyleProxy.m [hunk 29]
```diff
@@ -0,0 +1,25 @@
...
+#ifdef USE_TI_UIIOSPROGRESSBARSTYLE
+
+#import "TiUIiOSProgressBarStyleProxy.h"
+#import "TiBase.h"
+
+@implementation TiUIiOSProgressBarStyleProxy
+
+-(NSString*)apiName
+{
+    return @"Ti.UI.iPhone.ProgressBarStyle";
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSProgressBarStyleProxy.m]: Wrong apiName.

File: iphone/Classes/TiUIiOSProxy.m [hunk 30]
```diff
@@ -98,6 +98,13 @@
 
 @implementation TiUIiOSProxy
 
+#define FORGET_AND_RELEASE(x) \
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSProxy.m]: Why do we define that new macro here? Was it part of the TiUIiPhoneProxy.m ?

File: iphone/Classes/TiUIiOSRowAnimationStyleProxy.m [hunk 31]
```diff
@@ -0,0 +1,32 @@
...
+#ifdef USE_TI_UIIOSROWANIMATIONSTYLE
+
+#import "TiUIiOSRowAnimationStyleProxy.h"
+#import "TiBase.h"
+
+@implementation TiUIiOSRowAnimationStyleProxy
+
+-(NSString*)apiName
+{
+    return @"Ti.UI.iPhone.RowAnimationStyle";
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSRowAnimationStyleProxy.m]: Wrong apiName.

File: iphone/Classes/TiUIiOSScrollIndicatorStyleProxy.m [hunk 32]
```diff
@@ -0,0 +1,26 @@
...
+
+#import "TiUIiOSScrollIndicatorStyleProxy.h"
+
+#import "TiUtils.h"
+
+@implementation TiUIiOSScrollIndicatorStyleProxy
+
+-(NSString*)apiName
+{
+    return @"Ti.UI.iPhone.ScrollIndicatorStyle";
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSScrollIndicatorStyleProxy.m]: Wrong apiName.

File: iphone/Classes/TiUIiPhoneAlertDialogStyleProxy.m [hunk 33]
```diff
@@ -16,10 +16,10 @@ -(NSString*)apiName
     return @"Ti.UI.iPhone.AlertDialogStyle";
 }
 
-MAKE_SYSTEM_PROP(DEFAULT,                   UIAlertViewStyleDefault);
-MAKE_SYSTEM_PROP(PLAIN_TEXT_INPUT,          UIAlertViewStylePlainTextInput);
-MAKE_SYSTEM_PROP(SECURE_TEXT_INPUT,         UIAlertViewStyleSecureTextInput);
-MAKE_SYSTEM_PROP(LOGIN_AND_PASSWORD_INPUT,  UIAlertViewStyleLoginAndPasswordInput);
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(DEFAULT,                   UIAlertViewStyleDefault,       @"UI.iPhone.AlertDialogStyle.DEFAULT",@"6.0.0",@"UI.iOS.AlertDialogStyle.DEFAULT");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneAlertDialogStyleProxy.m]: Indentation looks weird, better indent it with normal space as done in newer API's.

File: iphone/Classes/TiUIiPhoneAnimationStyleProxy.m [hunk 34]
```diff
@@ -16,11 +16,11 @@ -(NSString*)apiName
     return @"Ti.UI.iPhone.AnimationStyle";
 }
 
-MAKE_SYSTEM_PROP(NONE,UIViewAnimationOptionTransitionNone);
-MAKE_SYSTEM_PROP(CURL_UP,UIViewAnimationOptionTransitionCurlUp);
-MAKE_SYSTEM_PROP(CURL_DOWN,UIViewAnimationOptionTransitionCurlDown);
-MAKE_SYSTEM_PROP(FLIP_FROM_LEFT,UIViewAnimationOptionTransitionFlipFromLeft);
-MAKE_SYSTEM_PROP(FLIP_FROM_RIGHT,UIViewAnimationOptionTransitionFlipFromRight);
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(NONE,              UIViewAnimationOptionTransitionNone,     @"UI.iPhone.AnimationStyle.NONE",@"6.0.0",@"UI.iOS.AnimationStyle.NONE");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneAnimationStyleProxy.m]: Same here (indentation).

File: iphone/Classes/TiUIiPhoneProxy.m [hunk 35]
```diff
@@ -226,10 +226,12 @@ -(NSString*)apiName
 
 BEGIN_UI_THREAD_PROTECTED_VALUE(statusBarHidden,NSNumber)
 result = [NSNumber numberWithBool:[[UIApplication sharedApplication] isStatusBarHidden]];
+DEPRECATED_REPLACED_REMOVED(@"UI.iPhone.statusBarHidden",@"3.1.3", @"6.0.0", @"UI.Window.fullscreen");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneProxy.m]: It should be on the `UI.Window` for longer as 6.0.0. Please consider, if this is necessary.

File: iphone/Classes/TiUIiPhoneProxy.m [hunk 36]
```diff
@@ -226,10 +226,12 @@ -(NSString*)apiName
...
 END_UI_THREAD_PROTECTED_VALUE(statusBarHidden)
 
 BEGIN_UI_THREAD_PROTECTED_VALUE(statusBarStyle,NSNumber)
 result = [NSNumber numberWithInt:[[UIApplication sharedApplication] statusBarStyle]];
+DEPRECATED_REPLACED_REMOVED(@"UI.iPhone.statusBarStyle",@"3.1.3", @"6.0.0", @"UI.Window.statusBarStyle");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneProxy.m]: I think this line would be printed everytime, something on the .iPhone. namespace is requested, because its not part of any method.

File: iphone/Classes/TiUIiPhoneProxy.m [hunk 37]
```diff
@@ -243,6 +245,7 @@ -(void)setAppBadge:(id)value
 	{
 		[[UIApplication sharedApplication] setApplicationIconBadgeNumber:[TiUtils intValue:value]];
 	}
+    DEPRECATED_REPLACED(@"UI.iPhone.appBadge", @"6.0.0", @"UI.iOS.appBadge");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneProxy.m]: Indentation

File: iphone/Classes/TiUIiPhoneProxy.m [hunk 38]
```diff
@@ -253,6 +256,7 @@ -(void)setAppSupportsShakeToEdit:(NSNumber *)shake
 {
 	ENSURE_UI_THREAD(setAppSupportsShakeToEdit,shake);
 	[[UIApplication sharedApplication] setApplicationSupportsShakeToEdit:[shake boolValue]];
+     DEPRECATED_REPLACED(@"UI.iPhone.appSupportsShakeToEdit", @"6.0.0", @"UI.iOS.appSupportsShakeToEdit");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneProxy.m]: Indentation

File: iphone/Classes/TiUIiPhoneSystemIconProxy.m [hunk 39]
```diff
@@ -16,19 +16,18 @@ -(NSString*)apiName
...
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(HISTORY,    UITabBarSystemItemHistory,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(MORE,       UITabBarSystemItemMore,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(MOST_RECENT,UITabBarSystemItemMostRecent,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(MOST_VIEWED,UITabBarSystemItemMostViewed,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(RECENTS,    UITabBarSystemItemRecents,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(SEARCH,     UITabBarSystemItemSearch,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(TOP_RATED,  UITabBarSystemItemTopRated,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneSystemIconProxy.m]: All properties wrong..**Don't copy and paste!!**. Also fix the indentation level.

File: iphone/Classes/TiUIiPhoneTableViewScrollPositionProxy.m [hunk 40]
```diff
@@ -17,10 +17,10 @@ -(NSString*)apiName
 }
 
 
-MAKE_SYSTEM_PROP(NONE,UITableViewScrollPositionNone);
-MAKE_SYSTEM_PROP(TOP,UITableViewScrollPositionTop);
-MAKE_SYSTEM_PROP(MIDDLE,UITableViewScrollPositionMiddle);
-MAKE_SYSTEM_PROP(BOTTOM,UITableViewScrollPositionBottom);
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(NONE,  UITableViewScrollPositionNone,@"UI.iPhone.ListViewScrollPosition.NONE",@"6.0.0",@"UI.iOS.ListViewScrollPosition.NONE");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(TOP,   UITableViewScrollPositionTop,@"UI.iPhone.ListViewScrollPosition.TOP",@"6.0.0",@"UI.iOS.ListViewScrollPosition.TOP");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneTableViewScrollPositionProxy.m]: Indentation

File: iphone/Classes/TiUIiPhoneTableViewStyleProxy.m [hunk 41]
```diff
@@ -15,8 +15,8 @@ -(NSString*)apiName
     return @"Ti.UI.iPhone.TableViewStyle";
 }
 
-MAKE_SYSTEM_PROP(PLAIN,UITableViewStylePlain);
-MAKE_SYSTEM_PROP(GROUPED,UITableViewStyleGrouped);
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(PLAIN,     UITableViewStylePlain,@"UI.iPhone.ListViewStyle.PLAIN",@"6.0.0",@"UI.iOS.ListViewStyle.PLAIN");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(GROUPED,   UITableViewStyleGrouped,@"UI.iPhone.ListViewStyle.GROUPED",@"6.0.0",@"UI.iOS.ListViewStyle.GROUPED");
```
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneTableViewStyleProxy.m]: Indentation

File: iphone/Classes/defines.h [hunk 42]
```diff
@@ -59,23 +59,23 @@
...
 #define USE_TI_UIIPHONETABLEVIEWCELLSELECTIONSTYLE
-#define USE_TI_UIIPHONEALERTDIALOGSTYLE
-#define USE_TI_UIIPHONELISTVIEWSTYLE
-#define USE_TI_UIIPHONELISTVIEWSCROLLPOSITION
-#define USE_TI_UIIPHONELISTVIEWCELLSELECTIONSTYLE
+#define USE_TI_UIIPHONEALERTDIALOGSTYLE//DEPRECATED
+#define USE_TI_UIIPHONELISTVIEWSTYLE//DEPRECATED
+#define USE_TI_UIIPHONELISTVIEWSCROLLPOSITION    //DEPRECATED
+#define USE_TI_UIIPHONELISTVIEWCELLSELECTIONSTYLE//DEPRECATED
 #define USE_TI_UIIPHONELISTVIEWSEPARATORSTYLE
```
Comment (hansemannn) [inline iphone/Classes/defines.h]: Thats deprecated as well, also remove the indentation in line 77

File: iphone/iphone/Titanium.plist [hunk 43]
```diff
@@ -52,7 +52,7 @@
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
 	<key>UIRequiresFullScreen</key>
-	<true/>
+	<false/>
```
Comment (hansemannn) [inline iphone/iphone/Titanium.plist]: Why did you change the value? Should be reverted to the original one.

**Comment by hansemannn** [comment]
Also please resolve the merge conflicts from current master before addressing any of the above comments. Thank you and don't give up! :-)

File: iphone/Classes/TiUIiPhoneProxy.m (hunk 35 above)
Comment (AngelkPetkov) [inline iphone/Classes/TiUIiPhoneProxy.m]: I don't understand what you mean for the first part ? Also the Macro above it is a method. So it only gets ran if the property is set in JS.

File: iphone/Classes/TiUIiPhoneTableViewScrollPositionProxy.m (hunk 40 above)
Comment (AngelkPetkov) [inline iphone/Classes/TiUIiPhoneTableViewScrollPositionProxy.m]: I did the indentation this way purposly so all the property namespaces line up with each other. It was like that for some of the old classes, also the new ones i added I've done the same. So I've tried to keep them the same where possible. For some of the deprecated classes/namespace i didn't do it as the deprecated message was too long and it made it impossible.

File: iphone/iphone/Titanium.plist (hunk 43 above)
Comment (AngelkPetkov) [inline iphone/iphone/Titanium.plist]: For testing. Will change back!

File: iphone/Classes/TiUIiPhoneProxy.m (hunk 35 above)
Comment (hansemannn) [inline iphone/Classes/TiUIiPhoneProxy.m]: Ok, the macros around it generate the method, get it. The property should not even be on the iPhone namespace anymore, since its **removed** since 3.1.3 (http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.iPhone-property-statusBarHidden). No one should use it anymore.

File: iphone/Classes/TiUIiOSProxy.h [hunk 44]
```diff
@@ -90,20 +137,79 @@
...
 
 @property(nonatomic,readonly) NSString* AD_SIZE_PORTRAIT;
 @property(nonatomic,readonly) NSString* AD_SIZE_LANDSCAPE;
-#endif
 
-#ifdef USE_TI_UIIOSBLURVIEW
-@property(nonatomic,readonly) NSNumber* BLUR_EFFECT_STYLE_EXTRA_LIGHT;
-@property(nonatomic,readonly) NSNumber* BLUR_EFFECT_STYLE_LIGHT;
-@property(nonatomic,readonly) NSNumber* BLUR_EFFECT_STYLE_DARK;
 #endif
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSProxy.h]: Why are the #endif and the new properties removed?

File: iphone/Classes/TiUIiOSProxy.h [hunk 45]
```diff
@@ -167,9 +273,6 @@
 #ifdef USE_TI_UIIOSMENUPOPUP
 -(id)createMenuPopup:(id)args;
 #endif
-#ifdef USE_TI_UIIOSBLURVIEW
--(id)createBlurView:(id)args;
-#endif
 #endif
```
Comment (hansemannn) [inline iphone/Classes/TiUIiOSProxy.h]: The changes should not be removed.

**Comment by AngelkPetkov** [comment]
https://github.com/appcelerator/titanium_mobile/pull/7662
